src/sidebar/util/postmessage-json-rpc.js

```diff
@@ -98,3 +98,23 @@ export function call(
       throw err;
     });
 }
+
+/**
+ * Send a JSON-RPC 2.0 notification request to another frame via `postMessage`.
+ * No response is expected.
+ *
+ * @param {Window} frame - Frame to send call to
+ * @param {string} origin - Origin filter for `window.postMessage` call
+ * @param {string} method - Name of the JSON-RPC method
+ * @param {unknown[]} params - Parameters of the JSON-RPC method
+ */
+function notify(frame, origin, method, params = []) {
+  const request = {
+    jsonrpc: '2.0',
+    method,
+    params,
+  };
+  frame.postMessage(request, origin);
+}
+
+export { call, notify };
```

Comment:
What is the reason for separating the export from the declaration here? Unless there is a strong need for it, we should put the `export` directly into the definition.

Reply:
I was just blindly following the export pattern that was already extant for this module. I can export each function individually; no problem.